packages/plugin-nested-docs/src/utilities/formatBreadcrumb.ts

```diff
@@ -19,8 +19,9 @@ export const formatBreadcrumb = (
   if (typeof pluginConfig?.generateLabel === 'function') {
     label = pluginConfig.generateLabel(docs, lastDoc)
   } else {
-    const useAsTitle = collection?.admin?.useAsTitle || 'id'
-    label = lastDoc[useAsTitle] as string
+    const title = lastDoc[collection?.admin?.useAsTitle || 'id']
```

Comment:
I dont think we need to optionally chain, or use `||` here, `id` is the default setting for useAsTitle.